Source/dead.h

```diff
@@ -5,16 +5,17 @@
  */
 #pragma once
 
+#include <array>
 #include <cstdint>
 
 #include "engine.h"
 
 namespace devilution {
 
-#define MAXDEAD 31
+static constexpr unsigned MAXDEAD = 31;
 
 struct DeadStruct {
-	uint8_t *_deadData[8];
+	std::array<uint8_t *, 8> _deadData;

```

Comment:
Oh, so this is what enabled the copy on assignment huh.

Reply:
Please resolve the comments when the question isn't relevant anymore.

Reply:
I cannot resolve my comments. Maybe I don't have enough permissions?

Reply:
ok, will look into that.